.mailmap

```diff
@@ -245,10 +265,13 @@ Irina Popa <irinagpopa@gmail.com>
 Ivan Ivaschenko <defuz.net@gmail.com>
 ivan tkachenko <me@ratijas.tk>
 J. J. Weber <jjweber@gmail.com>
-Jack Huey <jack.huey@umassmed.edu>
+Jack Huey <jack.huey@umassmed.edu> <jackh726@gmail.com>
 Jacob <jacob.macritchie@gmail.com>
+Jacob Hoffman-Andrews <rust@hoffman-andrews.com> <github@hoffman-andrews.com>
 Jacob Greenfield <xales@naveria.com>
 Jacob Pratt <jacob@jhpratt.dev> <the.z.cuber@gmail.com>
+Jake Goulding <jake.goulding@gmail.com> <jake.goulding@integer32.com>
+Jake Goulding <jake.goulding@gmail.com> <shepmaster@mac.com>
```

Comment:
```suggestion
Jake Goulding <jake.goulding@integer32.com> <jake.goulding@gmail.com> 
Jake Goulding <jake.goulding@integer32.com> <shepmaster@mac.com>
```

This is an attempt to make my Integer32 email the canonical one; I think I got the syntax right...

Reply:
That works, but I added another line to ensure that your name will always be mapped too (I don't know if that's necessary in this case or if your name is always consistent)